### PR TITLE
WIP - Docs improvements

### PR DESCRIPTION
**Pull request by pearofducks**

### CODE REVIEW

File: docs/client.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Given the following line now added to `eik.json`
 
 When we run `eik publish` any "bare imports" refering to either `lit-html` or `lodash` will be mapped to the URLs in our map.
 
-In this way, you can control which version of `react` or `lit-html` or `lodash` all your apps are using. In combination with package `alias` URLs, you have a powerful way to manage key shared dependencies for your apps in production without the need to redeploy or rebundle when a new version of a dependency is released.
+In this way, you can control which version of `react` or `lit-html` or `lodash` all your apps are using. In combination with package `alias` URLs, you have a powerful way to manage key shared dependencies for your apps in production without the need to redeploy or rebundle when a new version of a dependency are released.
```
Comment (digitalsadhu) [inline docs/client.md:151]: A new version of a dependency are released... *are* doesn't sound right to me, are you sure?

**Review by digitalsadhu — CHANGES_REQUESTED** [review]
Need to change the js.input field as noted as this is outdated.

File: docs/client.md (hunk 1 above)
Comment (pearofducks) [reply docs/client.md:151]: You're right, this is a bad change, will fix.

File: docs/client.md [hunk 2]
```diff
@@ -14,20 +14,20 @@ npm install -g @eik/cli
...
-See [the server docs](/docs/server)
+For the `server` property, you will need to have the address to a prepared Eik asset server.
+See [the server docs](/docs/server).
 
-Set the `files` property of `eik.json` with paths to client side
-asset files in your project relative to the `eik.json` file.
-Eg. if you have a `scripts.js` file in an assets directory, the `js.input` value will be `assets/scripts.js`
+Set the `files` property of `eik.json` with paths to client-side
+asset files in your project (relative to the `eik.json` file) -
+e.g. if you have a `scripts.js` file in an `assets` directory, the `js.input` value will be `assets/scripts.js`
```
[the remark this answers is not in the document]
Comment (pearofducks) [reply docs/client.md:30]: I assume `files` takes an array?

**Comment by pearofducks** [comment]
@digitalsadhu - have you guys discussed standardizing on a way to describe the Eik asset server in examples?

We aren't consistent in `http` vs `https`, and we use `myeikserver.com`, `<your asset server>`, and `assets.examplecdn.com` - and I'm just 2 docs in. :D

**Comment by digitalsadhu** [comment]
No we haven't. Doesn't matter much which but consistency would be nice. HTTPS is maybe best.

**Comment by digitalsadhu** [comment]
There's some weird out of date stuff showing up that I'm sure I fixed. I can take round on Monday and have a look. In any case the files stuff is documented correctly here. https://eik.dev/docs/client_eik_json

**Comment by pearofducks** [comment]
OK - I'll hold off on further edits then.

**Comment by trygve-lie** [comment]
> have you guys discussed standardizing on a way to reference the Eik asset server in examples?

Not really, but we have more or less on the agenda to do a "final go through" where the focus is to make sure we use the same therms, wording etc. I always find it good to get doc up to a certain level and then iterate over it to simplify and align stuff. Now we are also explaining what an Import Map is on 3 location and so on :)

**Pull request closed without merging**